core/src/test/java/org/mule/tck/testmodels/mule/TestMessageDispatcher.java

```diff
@@ -73,4 +95,10 @@ protected void doStop()
     {
         // no op
     }
+
+    @Override
+    protected boolean isSupportsNonBlocking()
```

Comment:
Is not better to have a NonBlockingTestMessageDispatcher extending from TestMessageDispatcher? (asking as this class is used in some tests that would expect this method to return false)